js/src/components/google-account-card/authorize-google-account-card.js

```diff
@@ -7,116 +7,60 @@ import { createInterpolateElement } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import { glaData } from '.~/constants';
 import AppButton from '.~/components/app-button';
-import AppDocumentationLink from '.~/components/app-documentation-link';
 import AccountCard, { APPEARANCE } from '.~/components/account-card';
-import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
-import useGoogleAuthorization from '.~/hooks/useGoogleAuthorization';
+import readMoreLink from './read-more-link';
+import useGoogleConnectFlow from './use-google-connect-flow';
 import './authorize-google-account-card.scss';
 
-const readMoreLink = (
-	<AppDocumentationLink
-		context="setup-mc-accounts"
-		linkId="required-google-permissions"
-		href="https://docs.woocommerce.com/document/google-listings-and-ads/#required-google-permissions"
-	/>
-);
-
 /**
  * Renders an AccountCard based on Google appearance for requesting Google authorization from user.
```

Comment:
💅 
Since I named the component starting with **Authorize**, to align with Google OAuth 2.0's "authorization" term. When asking users for authorizing GLA app, it covers both basic access to account information (`openid` and `userinfo.email`), as well as additional optional permission scopes (`content`, `siteverification.verify_only` and `adwords`).

If want to separate the requesting of Google authorization into two actions in GLA, I would suggest avoiding the combination of **Connect** and **Authorize** to prevent the ambiguity that **Authorize** on Google auth perspective can conceptually cover **Connect** on GLA perspective as well. But this component is going to be used for requesting more permission to users only.  Or, at least update its JSDoc description for clarifying the component usage.

A relevant context - https://github.com/woocommerce/google-listings-and-ads/pull/1052#discussion_r733333274

Reply:
Good point! I was thinking to rename the component to something like `RequestFullAccessGoogleAccountCard` to reflect the "Allow full access" button text in the UI and its responsibility, but I didn't do that because I thought **Authorize** word is fine enough. I will change the component name so that it is clearer and no ambiguity, and update the JSDoc description. 🙂  👍

Reply:
Done in [`3f75f9a` (#1074)](https://github.com/woocommerce/google-listings-and-ads/pull/1074/commits/3f75f9a911e1c33654d6f9d7fffba7646f0e2e71).